stix_shifter_modules/qradar/stix_translation/json/aql_flows_fields.json

```diff
@@ -50,7 +50,7 @@
     "sourcev6",
     "starttime",
     "endtime",
-    "BASE64(sourcepayload) as base64_source_payload",
-    "BASE64(destinationpayload) as base64_destination_payload"
+    "sourcepayload",
```

Comment:
Won't these need to use the UTF8 function like in the events fields? I think payload comes in the same for both events and flows. Maybe we should rename these as flowsourcepayload and flowdestinationpayload so we know where they are coming from when looking at the to-stix mapping.

Reply:
right. made that change